src/services/lowdbStorage.service.ts

```diff
@@ -26,10 +26,12 @@ export class LowdbStorageService implements StorageService {
         }
         try {
             this.db = lowdb(adapter);
+            fs.chmodSync(this.dataFilePath, "600");
```

Comment:
I don't think we want to be setting the permissions on every app init like this. What if the person wanted to manually change it to something else? Wouldn't this reset their changes every time? I would imagine we want to set this only once on file creation.

Reply:
Hello @kspearrin 
I've updated new commit as your suggestion.